src/sentry/monitors/constants.py

```diff
@@ -15,6 +15,9 @@
 # lowering this may invalidate monitors + block check-ins
 MAX_THRESHOLD = 720
 
+# minimum value in minutes for check-ins to close from in-progress
```

Comment:
Is this comment right? Isn't this for missed check-ins?

Reply:
ah. right